xray/src/bin/build_xray_tile.rs

```diff
@@ -31,12 +32,28 @@ fn parse_arguments() -> clap::ArgMatches<'static> {
             clap::Arg::with_name("resolution")
                 .help("Size of 1px in meters.")
                 .long("resolution")
-                .default_value("0.01"),
+                .default_value("0.05"),
```

Comment:
changing the default value is unrelated to this PR, right?

Reply:
Sorry, I missed this comment. yes it is, and I snuck a CI unbreaking change into this as well. I am sorry and will repent :(